components/producers/python-pip-safety/examples/requirements.txt

```diff
@@ -0,0 +1,20 @@
+Django==1.11.29
```

Comment:
same applies for this requirements file, please add a comment stating what it is for and mark the offending deps